### ISSUE TITLE
Global Let should allow capitalization of first charater

### ISSUE THREAD

**Issue by Marc-Hayes** [realm/SwiftLint#259]
I have found that it is common, though not truly consistent, to have the first letter of global lets to be capitalized rather then lowercase.

My personal use case for this is declaration of NSNotificationCenter names in the same style as Apple.

```swift
public let TopLevelFooBarNotificationIdentifier: String = "com.foo.bar"

public class FooBar {
...
NSNotificationCenter.defaultCenter().postNotificationName(TopLevelFooBarNotificationIdentifier, object: nil, userInfo: nil)
...
}

public class NotFooBar {
...
NSNotificationCenter.defaultCenter().addObserver(self, selector: "keyboardVisible", name: UIKeyboardWillShowNotification, object: nil)
NSNotificationCenter.defaultCenter().addObserver(self, selector: "doTheThing:", name: TopLevelFooBarNotificationIdentifier, object: nil)
...
}
```

**Comment by scottrhoyt** [comment]
I agree that this has also come up for me, though I can't decide how I feel about it. I've used capitals for the global cases just to be consistent with some of Apple's global constants. But Apple is also inconsistent. Sometimes it's the capital lettering, other times its the c-style `kConstant`, and so on. So I also think I'd be just as happy adhering to a more consistent standard.

**Comment by scottrhoyt** [comment]
A workaround for this is now possible via the `excluded` parameter for naming rules.

**Comment by Marc-Hayes** [comment]
Thank you for letting me know!
